### PR TITLE
feat: 7398 edit page wizard

### PR DESCRIPTION
**Pull request by joshbalfour**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #7398

### CODE REVIEW

File: packages/app-builder/src/components/ui/toolbox.tsx [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 import Link from './user/link'
-import Table from './user/table'
-import Form from './user/form'
-import Info from './user/info'
+// import Table from './user/table'
+// import Form from './user/form'
+// import { TableIcon } from '../icons/table'
+// import { FormIcon } from '../icons/form'
+// import Info from './user/info'
+// import SquareSvg from '../icons/square'
```
Comment (bashleigh) [inline packages/app-builder/src/components/ui/toolbox.tsx:16]: still want this?

File: packages/app-builder/src/components/ui/toolbox.tsx [hunk 2]
```diff
@@ -113,15 +113,15 @@ export const Toolbox = ({ create }: { create: (ref: HTMLElement, ele: any) => vo
...
           name="Info"
           ref={(ref: HTMLElement) => ref && create(ref, <Element is={Info} padding={12} width={12} />)}
         >
           <Item>
             <SquareSvg />
           </Item>
-        </ToolboxItem>
-        <ToolboxItem
+        </ToolboxItem> */}
+        {/* <ToolboxItem
```
Comment (bashleigh) [inline packages/app-builder/src/components/ui/toolbox.tsx:124]: and these?

File: packages/app-builder/src/components/ui/toolbar/index.tsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ const ToolbarContainer = styled.div`
 
     label {
       margin-right: 31px;
-      min-width: 85px;
+      /* min-width: 85px; */
```
Comment (willmcvay) [inline packages/app-builder/src/components/ui/toolbar/index.tsx:31]: You still need?

File: packages/app-builder/src/components/ui/toolbox.tsx [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 import Text from './user/text'
 import { cx } from '@linaria/core'
 import { styled } from '@linaria/react'
 import { block, cursorMove, transition } from './styles'
 import { elFlex, elFlexAlignStart, elFlexJustifyStart, elFlexRow, elFlexWrap, elM3, elW12 } from '@reapit/elements'
 import Link from './user/link'
-import Table from './user/table'
-import Form from './user/form'
-import Info from './user/info'
+// import Table from './user/table'
```
Comment (willmcvay) [inline packages/app-builder/src/components/ui/toolbox.tsx:11]: And the stuff in this file?

**Review by willmcvay — APPROVED** [review]

**Review by bashleigh — APPROVED** [review]

**Pull request merged**